yoda/src/main/java/life/jukebox/App.java

```diff
@@ -0,0 +1,9 @@
+package life.jukebox;
+
+public class App 
+{
+    public static void main( String[] args )
```

Comment:
Please adhere to project standards of using 2 spaces for every tab. You can change this in your ~/.vimrc (from 4 spaces to 2 tabs)